### PR TITLE
[OMP][main] #8093 UserGroupDAO, UserGroupAssignmentDAO functions replaced

### PR DESCRIPTION
**Pull request by defstat**

### CODE REVIEW

File: classes/submission/Submission.php [hunk 1]
```diff
@@ -299,14 +299,7 @@ public function getCoverImageAltText()
...
         if ($this->getWorkType() != self::WORK_TYPE_EDITED_VOLUME) {
-            $userGroupIds = array_map(function ($author) {
-                return $author->getData('userGroupId');
-            }, Repo::author()->getSubmissionAuthors($this, true)->toArray());
-
-            $userGroups = array_map(function ($userGroupId) {
-                return DAORegistry::getDAO('UserGroupDAO')->getbyId($userGroupId);
-            }, array_unique($userGroupIds));
-            return $this->getCurrentPublication()->getAuthorString($userGroups);
+            return $this->getAuthorString($preferred);
```
Comment (asmecher) [inline classes/submission/Submission.php:302]: `PKPSubmission::getAuthorString` is deprecated.

File: plugins/importexport/csv/CSVImportExportPlugin.php [hunk 2]
```diff
@@ -134,7 +133,12 @@ public function executeCLI($scriptName, &$args)
                     if (!is_array($supportedLocales) || count($supportedLocales) < 1) {
                         $supportedLocales = [$press->getPrimaryLocale()];
                     }
-                    $authorGroup = $userGroupDao->getDefaultByRoleId($press->getId(), Role::ROLE_ID_AUTHOR);
+                    $authorGroup = Repo::userGroup()->getMany(
```
Comment (asmecher) [inline plugins/importexport/csv/CSVImportExportPlugin.php:136]: This needs to be updated for https://github.com/pkp/pkp-lib/issues/8092.

**Review by asmecher — CHANGES_REQUESTED** [review]

File: classes/submission/Submission.php (hunk 1 above)
Comment (defstat) [reply classes/submission/Submission.php]: fixed (I think)

File: plugins/importexport/csv/CSVImportExportPlugin.php (hunk 2 above)
Comment (defstat) [reply plugins/importexport/csv/CSVImportExportPlugin.php]: fixed

**Comment by asmecher** [comment]
Cherry-picked manually.

**Pull request closed without merging**